pkg/http/api_directmessaging.go

```diff
@@ -292,3 +313,14 @@ type invokeError struct {
 func (ie invokeError) Error() string {
 	return fmt.Sprintf("invokeError (statusCode='%d') msg='%v'", ie.statusCode, string(ie.msg))
 }
+
+type codeError struct {
+	statusCode  int
+	msg         []byte
+	headers     invokev1.DaprInternalMetadata
+	contentType string
+}
+
+func (ce codeError) Error() string {
+	return fmt.Sprintf("received non-successful status code in response: statusCode='%d' msg='%v'", ce.statusCode, string(ce.msg))
```

Comment:
On a second thought... Let's not log the message as it may contain PII. Sorry for the back-and-forth.

```suggestion
	return fmt.Sprintf("received non-successful status code in response: %d", ce.statusCode)
```